### PR TITLE
the function is ready

### PR DESCRIPTION
**Pull request by galamo**
this is my function ...

### CODE REVIEW

**Comment by roiroi123** [comment]
ok

**Comment by TomGagement** [comment]
looks good

**Comment by jonivid** [comment]
looks good

**Comment by mikelbts89** [comment]
👍

File: index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+function sumNumbers(number){
+    if(typeof number !== 'number') return;
+    let sum = 0
```
Comment (galamo) [inline index.js:3]: l

**Pull request merged**